src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -2321,7 +2321,8 @@ else if (cmd.getName().equalsIgnoreCase("adjustbonusclaimblocks"))
             }
             catch (NumberFormatException numberFormatException)
             {
-                return false;  //causes usage to be displayed
+                GriefPrevention.sendMessage(player, TextMode.Err, Messages.InvalidNumber);
+                return true;
```

Comment:
I think this may be ok, as long as this returns `false` so it can show usage message in case there's something else wrong with the format of this command, or the user thought it's some other command.